### ISSUE TITLE
path error

### ISSUE THREAD

**Issue by Simon-He95** [infi-pc/locatorjs#46]
**Describe the bug**
I'm a Windows user, but the terminal I use is wsl. My project is in a Windows environment, 
the directory of the runtime and the path of my real project will be different.
so when I use alt + click will find wsl path, so it won't find the file under windows project.
![image](https://user-images.githubusercontent.com/57086651/183085044-8baca524-5616-4b06-9c29-28bc4c46f2c0.png)

** Browser **
 - Chrome
 - Firefox

** OS **
- Windows

** IDE **
- VSCode

**To Reproduce**
1. Click on '....'
2. See error 
![7067510f942471d75bf6b29da1d3321](https://user-images.githubusercontent.com/57086651/183087079-ca3df7f7-d9e6-4dc3-8726-e205a0857e4b.jpg)

**Expected behavior**
 Click on can show Correct location under windows project.

**Comment by infi-pc** [comment]
Hi @Simon-He95, it sounds similar to this: https://github.com/infi-pc/locatorjs/issues/30
tldr: you can change the editor link to custom and modify it to something similar to this: `vscode://vscode-remote/wsl+Ubuntu/home/..` 
will it help?

I will try to make a better WSL support in the future, but now I hope it is solvable by changing the editor path.

**Comment by Simon-He95** [comment]
Thanks for your advice, I have tried it, but it doesn't work, it still can't find the directory for the project. Have I overlooked anything?
![image](https://user-images.githubusercontent.com/57086651/183137840-96a3c055-12f1-42c1-8853-1c6568283ee5.png)

Error message:
![image](https://user-images.githubusercontent.com/57086651/183138080-5c73bfec-99d3-4986-ad3e-f456e7d8e9c8.png)

**Comment by infi-pc** [comment]
🤔 you have to change the template in the input field. But I can't say exactly what because I don't know WSL very much. And I can't try it currently on Windows machine. 

I am guessing it will be something like this: `vscode://vscode-remote/wsl+Ubuntu/${projectPath}${filePath}:$(line}:${column}` or if you have it in your windows file system there may be a way how to make the local path from the original path: I would try adding `..` to "remove" the `/mnt/` from your path so there will stay only this path `/d/GitHub/vitesse-lite-react/` which I am guessing is your local path. The final template would be: `vscode://file/../${projectPath}${filePath}:$(line}:${column}`

**Comment by Simon-He95** [comment]
Hi, @infi-pc thank you for your help, I use local so I won't expect to open remote wsl vscode, I try to use the second way that `vscode://file/../${projectPath}${filePath}:$(line}:${column}` to adding .., but it can't remove /mnt/. I think just remove the /mnt/ and it will be able to find my local project path correctly.Do you have any good ideas ?

**Comment by infi-pc** [comment]
Hey, I just added RegEx transformation functionality to help with this and similar use cases. 
You can replace anything in the original path with regexp.

![CleanShot 2022-09-30 at 12 09 45@2x](https://user-images.githubusercontent.com/2264725/193248020-c37d5b41-d966-4269-b85f-c0c14e276235.png)
